app/main/views/briefs.py

```diff
@@ -81,6 +83,10 @@ def ask_brief_clarification_question(brief_id):
 def start_brief_response(brief_id):
     brief = get_brief(data_api_client, brief_id, allowed_statuses=['live'])
 
+    if not (datetime.strptime(current_app.config['FEATURE_FLAGS_NEW_SUPPLIER_FLOW'], "%Y-%m-%d")
+            <= datetime.strptime(brief['publishedAt'][0:10], "%Y-%m-%d")):
```

Comment:
Cutting a date_string seems like a slightly brittle way of zero'ing the rest of it beyond year, month and day.

I say that not knowing how solid the format is and knowing little about best practice in Python so take this with a pinch of salt.